src/matchmaking/player.hpp

```diff
@@ -34,10 +36,6 @@ class Player {
     // Returns false if the time control has been exceeded.
     [[nodiscard]] bool updateTime(const int64_t elapsed_millis) {
         auto &tc = time_control_;
-        if (tc.time_left == 0) {
-            return true;
```

Comment:
i right now wonder why i had return true here instead of false

Reply:
i think you added it in so that when the user is running a fixed depth/node test it wouldnt immediately timeloss

Reply:
As thanks that sounds reasonable, I was just wondering if this was actually helping in avoiding timelosses. I will later check since when it’s there